docs/scale-cluster.rst

```diff
@@ -44,32 +44,39 @@ dropdown menu at the top left to see the desired projects.
 In the menu on the left hand you should see below the divider line an icon or
 icons representing all clusters associated with the currently selected project.
 Here, find the correct cluster and click on the icon. It will expand and show
-links for three pages: *Overview*, *Metrics* and *Settings*.
+links for four pages: *Overview*, *Backups*, *Metrics* and *Preferences*.
 
 .. image:: _assets/img/cluster-dropdown.png
    :alt: Cloud Console projects cluster selection
 
-Click on *Settings*. This will take you to the Cluster Settings page, where
-you can see the current scale unit of the cluster, as well as an information
-box for the current cluster plan and tier. To scale the cluster up or down
-within the permitted range, simply click the *Edit scale unit* button at the
-top right.
+Click on *Preferences*. This will take you to cluster access page, where you
+can see and edit your credentials. You can also enable deletion protection 
+of your cluster and create a whitelist of IP addresses or CIDR blocks that are
+allowed to connect to the cluster. 
 
-.. image:: _assets/img/cluster-settings.png
-   :alt: Cloud Console cluster settings
+.. image:: _assets/img/cluster-access.png
+   :alt: Cloud Console cluster access settings
+
+To edit the cluster scale, click on *Cluster scale* in the top bar. This will take
+you to a page where you can see an overview of your Cluster plan as well as 
+current cluster configuration.
+
+.. image:: _assets/img/cluster-preferences.png
+   :alt: Cloud Console cluster preferences
 
 
 .. _scale-cluster-instructions:
 
 Scale a cluster
 ===============
 
-Once arrived at the Cluster Settings page, you can scale the cluster you
-selected in three steps. First, click the *Edit scale unit* button in the top
-right.
+Once you arrive at the Cluster scale page, you can scale the cluster you
```

Comment:
I suppose we should be consistent about how to refer to the docs tabs/pages/etc... I think in some of these older files I didn't quite do that (e.g., see the bit on Cluster Preferences above, which is a page rather than a tab - what to do?) I think the Console overview in the Reference has an implicit standardization that we should use, might be worth checking out to see if that makes sense to you